localstack/services/lambda_/invocation/docker_runtime_executor.py

```diff
@@ -405,6 +409,7 @@ def get_logs(self) -> str:
     @classmethod
     def prepare_version(cls, function_version: FunctionVersion) -> None:
         lambda_hooks.prepare_docker_executor.run(function_version)
+        get_runtime_client_path()
```

Comment:
We might want to add a comment there explaining why we do this here

Reply:
agree, done

Reply:
sidenote: as long as we respect the function states and do not perform an invoke while the function is still pending, we should be protected from race conditions where two downloads are triggered simultaneously. This is currently a non-issue because the runtime-init is baked into the image.